k8s/mongodb/container/mongod_entrypoint.bash

```diff
@@ -65,9 +65,8 @@ if [[ -z "${REPLICA_SET_NAME:?REPLICA_SET_NAME not specified. Exiting!}" || \
     -z "${MONGODB_IP:?MONGODB_IP not specified. Exiting!}" || \
     -z "${MONGODB_KEY_FILE_PATH:?MONGODB_KEY_FILE_PATH not specified. Exiting!}" || \
     -z "${MONGODB_CA_FILE_PATH:?MONGODB_CA_FILE_PATH not specified. Exiting!}" || \
-    -z "${MONGODB_CRL_FILE_PATH:?MONGODB_CRL_FILE_PATH not specified. Exiting!}" || \
-    -z ${STORAGE_ENGINE_CACHE_SIZE:=''} ]] ; then
-    #-z "${MONGODB_KEY_FILE_PASSWORD:?MongoDB Key File Password not specified. Exiting!}" || \
+    -z "${MONGODB_CRL_FILE_PATH:?MONGODB_CRL_FILE_PATH not specified. Exiting!}" ]] ; then
```

Comment:
Maybe you could include a comment line so a careful reader can be reassured that you purposefully didn't include `STORAGE_ENGINE_CACHE_SIZE` (or others?). Maybe even say why.

Reply:
Done

Reply:
:+1: